### PR TITLE
Wikidpad: init at WikidPad-2-3-beta13_01

### PR DESCRIPTION
**Pull request by deliciouslytyped**
~~https://github.com/NixOS/nixpkgs/issues/55681~~
Edit: superseded by Superseded by https://github.com/NixOS/nixpkgs/issues/55681, will close after merged.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by dtzWill** [comment]
Would it make sense to use newer 2.3 release, such as rc2?

**Comment by deliciouslytyped** [comment]
I'm actually not sure, I just grabbed the first thing that wasn't marked as pre-release: https://github.com/WikidPad/WikidPad/releases . Actually, I guess I should add a reference to the github page somewhere in here.

File: pkgs/applications/office/wikidpad/2.3.nix [hunk 1]
```diff
@@ -0,0 +1,38 @@
+#References
+# http://trac.wikidpad2.webfactional.com/wiki/InstallLinux
+
+{stdenv, lib, fetchFromGitHub, python27, makeWrapper, wrapGAppsHook}:
+let
+  fetchGitHubJSON = file: fetchFromGitHub { inherit (lib.importJSON file) owner repo rev sha256; };
```
Comment (ryantm) [inline pkgs/applications/office/wikidpad/2.3.nix]: To me, it seems excessive to have this JSON file and all this extra machinery to fetch the source when we are only getting one revision. Can we make this just be a normal fetchFromGitHub, like most of nixpkgs?

File: pkgs/applications/office/wikidpad/2.3.nix [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      '';
+
+    meta = {
+      description = ''
+        wikidPad is a Wiki-like notebook for storing your thoughts, ideas,
+        todo lists, contacts, or anything else you can think of to write down.
+        '';
+      homepage = "http://wikidpad.sourceforge.net/";
+      license = with lib.licenses; [ bsd2 asl20 ];
+      };
```
Comment (ryantm) [inline pkgs/applications/office/wikidpad/2.3.nix]: Please add yourself as a maintainer in a separate commit.

**Review by ryantm — CHANGES_REQUESTED** [review]
Also, it would be good to rename the file to default.nix like usual.

**Pull request closed without merging**